tasklist/client/src/Tasks/CollapsiblePanel/ControlledNavLink/index.tsx

```diff
@@ -38,6 +42,7 @@ const ControlledNavLink: React.FC<Props> = ({
           className,
         )
       }
+      title={children.length > 17 ? children : undefined}
```

Comment:
What's this doing exactly?

Reply:
@Skaiir  It's showing a native browser tooltip when the text cuts with an ellipsis